README.md

```diff
@@ -1 +1,312 @@
+
+### Deploying to AWS Lambda
+
+```bash
+$ aws lambda create-function --function-name processImages --zip-file fileb://aws-ballerina-lambda-functions.zip --handler image-text-processor.processImages --runtime provided --role arn:aws:iam::908363916138:role/lambda-role --timeout 10 --memory-size 1024
+{
+    "TracingConfig": {
+        "Mode": "PassThrough"
+    }, 
+    "CodeSha256": "lwg8/90iroOzc5fFYU4Fp9r9CpMnsWgx2xOPekldHdA=", 
+    "FunctionName": "processImages", 
+    "CodeSize": 41462, 
+    "RevisionId": "ba962bb9-976e-47f1-9349-cc79319cc30d", 
+    "MemorySize": 1024, 
+    "FunctionArn": "arn:aws:lambda:us-west-2:908363916138:function:processImages", 
+    "Version": "$LATEST", 
+    "Role": "arn:aws:iam::908363916138:role/lambda-role", 
+    "Timeout": 10, 
+    "LastModified": "2019-02-02T00:28:02.658+0000", 
+    "Handler": "image-text-processor.processImages", 
+    "Runtime": "provided", 
+    "Description": ""
+}
+
+$ aws lambda update-function-configuration --function-name processImages --layers arn:aws:lambda:us-west-2:908363916138:layer:ballerina-0_990_3-runtime:19
+{
+    "Layers": [
+        {
+            "CodeSize": 175675006, 
+            "Arn": "arn:aws:lambda:us-west-2:908363916138:layer:ballerina-0_990_3-runtime:19"
+        }
+    ], 
+    "FunctionName": "processImages", 
+    "LastModified": "2019-02-02T00:30:55.803+0000", 
+    "RevisionId": "7637cae8-703b-489a-87af-ecdae62b1816", 
+    "MemorySize": 1024, 
+    "Version": "$LATEST", 
+    "Role": "arn:aws:iam::908363916138:role/lambda-role", 
+    "Timeout": 10, 
+    "Runtime": "provided", 
+    "TracingConfig": {
+        "Mode": "PassThrough"
+    }, 
+    "CodeSha256": "lwg8/90iroOzc5fFYU4Fp9r9CpMnsWgx2xOPekldHdA=", 
+    "Description": "", 
+    "CodeSize": 41462, 
+    "FunctionArn": "arn:aws:lambda:us-west-2:908363916138:function:processImages", 
+    "Handler": "image-text-processor.processImages"
+}
+```
+
+### Updating Connector Credential Environment Variables
+
+The credentials for the Amazon Rekognize and GMail connectors can be provided by setting environment variables for the target AWS Lambda function. The following command demonstrates this action.
+
+```bash
+aws lambda update-function-configuration --function-name processImages --environment "Variables={AMAZON_ACCESS_KEY=AXXXJKKK,AMAZON_SECRET_KEY=/XXXAAAA,GMAIL_CLIENTID=AAAXXXX.apps.googleusercontent.com,GMAIL_CLIENTSECRET=XXXXAAAAXXXXBBBBBBBXXXXX,GMAIL_REFRESHTOKEN=XXXXOFFXXXXX,GMAIL_ACCESSTOKEN=XXXX.XXX}"
+```
+Fill in the environment variables values with your own API credentials. The above updates can also be done using the AWS Lambda web console, by navigating to the respective function's configuration page. 
```

Comment:
```suggestion
>*Tip:**Replace the values of the environment variable with your own API credentials. Alternatively, you can do this via the AWS Lambda web Console by navigating to the configuration page of the respective function. 
```

Reply:
This is more of a "Note", will add your suggestion as a "Note" section.